.github/workflows/ci.yml

```diff
@@ -189,7 +189,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-22.04, ubuntu-24.04, ubuntu-24.04-arm, macos-13, windows-2022]
-        go-version: ["1.23.12", "1.24.6"]
+        go-version: ["1.24.6", "1.25.0"]
```

Comment:
shouldn't CI continue to test the minimum version?

Reply:
We are changing the minimum version to 1.24.x since 1.23 is now EOL.  Should we update the go.mod too?

Reply:
Why are we changing minimum versions in release branches?

Reply:
Since we dont want to keep an EOL go version as the minimal one. 

Also,  2.1 has almost another year of support, if we decide not to change the minimum version in release branches, 1.23 will have to be maintained in CI for a long time.